pymech/exadata.py

```diff
@@ -158,6 +158,10 @@ def lims(self):
 		return datalims(self.var, self.elem)
 
 	def check_connectivity(self):
```

Comment:
So I'm a bit embarrassed because I realized while writing the .re2 functions that connectivity information is not even read by Nek (it's not present in .re2 files and may be invalid in .rea files), so this function is kind of useless except for 'P' (periodic) conditions. But we can leave it like this for now.

Reply:
Thanks for clarifying that. Nek keeps surprising us, sometimes for the wrong reason :)